components/suggestion/at_mention_provider.jsx

```diff
@@ -152,9 +154,9 @@ export default class AtMentionProvider extends Provider {
                 }
 
                 let users = members.concat(specialMentions).concat(nonmembers);
-                const me = UserStore.getCurrentUser();
+                const currentUserId = getCurrentUserId(store.getState());
```

Comment:
Shouldn't we be looking at getting rid of global store access with this? Or is that coming later?

Reply:
Eventually yes I'd agree but two things:
1) Need to figure out the proper patterns for how to do this properly
2) This PR (and the ones like it) is meant to be narrowly focused on just removing flux store usages